stack/rest/src/test/java/org/apache/usergrid/rest/management/RegistrationIT.java

```diff
@@ -250,48 +274,56 @@ public void postAddToOrganization() throws Exception {
     @Test
     public void addNewAdminUserWithNoPwdToOrganization() throws Exception {
 
-        Map<String, String> originalProperties = getRemoteTestProperties();
+        Map<String, Object> originalProperties = getRemoteTestProperties();
 
         try {
             Mailbox.clearAll();
-            setTestProperty( PROPERTIES_SYSADMIN_APPROVES_ADMIN_USERS, "false" );
-            setTestProperty( PROPERTIES_SYSADMIN_APPROVES_ORGANIZATIONS, "false" );
-            setTestProperty( PROPERTIES_ADMIN_USERS_REQUIRE_CONFIRMATION, "false" );
-            setTestProperty( PROPERTIES_SYSADMIN_EMAIL, "sysadmin-1@mockserver.com" );
+            setTestProperty(PROPERTIES_SYSADMIN_APPROVES_ADMIN_USERS, "false");
+            setTestProperty(PROPERTIES_SYSADMIN_APPROVES_ORGANIZATIONS, "false");
+            setTestProperty(PROPERTIES_ADMIN_USERS_REQUIRE_CONFIRMATION, "false");
+            setTestProperty(PROPERTIES_SYSADMIN_EMAIL, "sysadmin-1@mockserver.com");
 
             // this should send resetpwd  link in email to newly added org admin user(that did not exist
             ///in usergrid) and "User Invited To Organization" email
-            String adminToken = adminToken();
-            JsonNode node = postAddAdminToOrg( "test-organization", "test-admin-nopwd@mockserver.com", "", adminToken );
-            String uuid = node.get( "data" ).get( "user" ).get( "uuid" ).textValue();
-            UUID userId = UUID.fromString( uuid );
+            Entity node = postAddAdminToOrg("test-organization", "test-admin-nopwd@mockserver.com", "", getAdminToken());
```

Comment:
Probably don't want to use the default organization because we might have the test-organization disabled in the properties file.